skydb/skydb.go

```diff
@@ -88,6 +87,30 @@ func (db SkyDB) Write(data []byte, dataKey crypto.Hash, rev uint64) error {
 	return nil
 }
 
+// EntropyFromEnv returns the configured value of the SKYDB_ENTROPY environment
+// variable or an error.
+func EntropyFromEnv() ([32]byte, error) {
+	var e [32]byte
+	eStr := os.Getenv("SKYDB_ENTROPY")
+	if eStr == "" {
+		return e, errors.New("missing or empty SKYDB_ENTROPY environment variable. it needs to contain 32 bytes of entropy.")
+	}
+	eBytes, err := base64.StdEncoding.DecodeString(eStr)
+	if err != nil || len(eBytes) != 32 {
+		errmsg := fmt.Sprintf("invalid SKYDB_ENTROPY environment variable. it needs to contain 32 bytes of entropy. error: %v", err)
+		return e, errors.New(errmsg)
+	}
+	copy(e[:], eBytes)
+	return e, nil
+}
+
+// skynetFilePath returns a path that a Skyfile can be uploaded to. The path is
+// based on the provided dataKey.
+func skynetFilePath(dataKey crypto.Hash) (sp skymodules.SiaPath) {
```

Comment:
Are you trying to avoid dependencies? If not, you could use `skymodules.RandomSkynetPath`.

Reply:
The idea was to reuse the same siapath as before based on this discussion: https://github.com/SkynetLabs/certmagic-storage-skydb/pull/1/#discussion_r645636229